### PR TITLE
Imp product by supplier

### PR DESCRIPTION
**Pull request by hurrinico**
moved from:
https://code.launchpad.net/~agilebg/purchase-wkfl/imp_product_by_supplier/+merge/225197

### CODE REVIEW

**Comment by bwrsandman** [comment]
LGTM :+1:

File: product_by_supplier/product.py [hunk 1]
```diff
@@ -29,7 +31,7 @@ def _product_available(
             self, cr, uid, ids, field_names=None, arg=False, context=None):
         context = context or {}
         res = {}
-        product_obj = self.pool.get('product.product')
+        product_obj = self.pool.['product.product']
```
Comment (gurneyalex) [inline product_by_supplier/product.py]: syntax error -> needs fixing.

Comment (hurrinico) [inline product_by_supplier/product.py]: Sorry, i'have made this fix for the problem

**Comment by eLBati** [comment]
:+1:

**Comment by pedrobaeza** [comment]
Travis errors are unrelated, so :+1:

**Pull request merged**